app/assets/stylesheets/common/components/bookmark-menu.scss

```diff
@@ -0,0 +1,113 @@
+@keyframes slide {
+  0% {
+  }
+  100% {
+    transform: translateY(100%) translateX(-50%);
+    opacity: 0;
+  }
+}
+
+.bookmark-menu-content {
+  &__notice {
+    position: absolute;
+    top: -1.5rem;
+    left: 50%;
+    transform: translateX(-50%);
+    color: var(--tertiary);
+    padding: 0.25rem 0.5rem;
+    white-space: nowrap;
+    opacity: 1;
+
+    &.-slide-out {
+      animation: slide 1s cubic-bezier(0, 0, 0, 2) forwards;
+    }
+  }
+
+  .bookmark-menu__body {
+    background: var(--secondary);
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    color: var(--primary);
+
+    .bookmark-menu__actions {
+      margin: 0;
+      padding: 0;
+      list-style: none;
+    }
+  }
+
+  .bookmark-menu {
+    &__text {
+      display: flex;
+      align-items: left;
+    }
+
+    &__row {
+      border-bottom: 1px solid var(--primary-low);
+      width: 100%;
+      display: flex;
+      align-items: left;
+
+      &:hover,
+      &:focus {
+        background: var(--tertiary-very-low) !important;
```

Comment:
I'm surprised we need so many !important

Reply:
I got rid of them, didn't seem to affect anything